src/mpid/ch4/src/ch4r_win.h

```diff
@@ -308,6 +308,17 @@ static inline int MPIDI_CH4R_mpi_win_set_info(MPIR_Win * win, MPIR_Info * info)
     if (mpi_errno != MPI_SUCCESS)
         MPIR_ERR_POP(mpi_errno);
 
+    /* netmod or shmmod can update its internal setting based on user updated info. */
+    mpi_errno = MPIDI_NM_mpi_win_set_info_hook(win);
+    if (mpi_errno != MPI_SUCCESS)
+        MPIR_ERR_POP(mpi_errno);
+
```

Comment:
Is it possible to have two types of hints for RMA window: hints that can be set only at window creation time, and hints that can be changed later, with use of `MPI_Win_set_info`?

Reply:
We can distinguish some of the RMA hints in such way, e.g., same_size, same_disp_unit. But we do not have any real use case now.